mango-cli/src/main/webapp/WEB-INF/layouts/overall.ssp

```diff
@@ -105,6 +104,13 @@ function home(dictionary, regions){
 				.style("opacity", 0);
 		});
 
+	chart
+		.append("text")
+		.attr("x", "50%")
+		.attr("y",function(d,i){return height*i})
+		.attr("dy", "1.1em")
+		.text(function(d) { return d.name; });
+
```

Comment:
To get around the issue of having the label covered up, I believe you can move this to after the rendering of regions.  Regions is printed here: https://github.com/Georgehe4/mango/blob/54b0217c427c19d71d46a56dddf5194cb366490d/mango-cli/src/main/webapp/WEB-INF/layouts/overall.ssp#L114